RELEASE_NOTES.md

```diff
@@ -0,0 +1,112 @@
+
+-   OPAE Software Development Kit (OPAE SDK),
+
+-   OPAE Linux driver for Intel(R) Xeon(R) CPU with Integrated FPGAs and Intel(R) PAC with Arria(R) 10 GX FPGA
+
+-   Basic Building Block (BBB) library for accelerating AFU
+    development (not part of this release, but pre-release code is
+    available on GitHub: \[https://github.com/OPAE/intel-fpga-bbb]
+
+OPAE is under active development to extend to more hardware platforms, as well as to build up the software stack with additional abstractions to enable more software developers.
+
+OPAE SDK is a collection of libraries and tools to facilitate the development of software applications and accelerators using OPAE. It provides a library implementing the OPAE C API for presenting a streamlined and easy-to-use interface for software applications to discover, access, and manage FPGA devices and accelerators using the OPAE software stack. The OPAE SDK also includes the AFU Simulation Environment (ASE) for end-to-end simulation of accelerator RTL together with software applications using the OPAE C API.
+
+OPAE\'s goal is to accelerate FPGA adoption. It is a community effort to simplify the development and deployment of FPGA applications, so we explicitly welcome discussions and contributions! The OPAE SDK source, unless otherwise noted, is released under a BSD 3-clause license.
+
+More information about OPAE can be found
+at http://01.org/OPAE.
+
+Open Programmable Acceleration Engine (OPAE) 1.1.0 Release Notes
+-----------------------------------------------------------------
+
+This document provides the Release Notes for the Open Programmable
+Acceleration Engine (OPAE) 1.1.0 release.
+
+System Compatibility
+--------------------
+
+-   Hardware: tightly coupled FPGA products and programmable FPGA
+    acceleration cards for Intel(R) Xeon(R) processors (to be released):
+    - Intel(R) PAC with Arria(R) 10 GX FPGA (PCI ID: 0x09c4) FIM version 1.0.3 (1.0 Production)
+    - Intel(R) Xeon with Integrated FPGA (PCI ID: 0xbcc0) FIM version 6.4.0
+
+-   Operating System: tested on Red Hat Enterprise Linux 7.3, SUSE SLE 12 SP3 and CentOS
+    7.4, with Linux kernels 3.10 through 4.7
+
+Major Changes from 1.0.0 to 1.1.0
+----------------------------------
+
+
+
+- Updated command line options for OPAE tools for a consistent user interface
+- Added two new language bindings
+  - A C++ Core API that is interoperable with the OPAE C API
+  - A Python API which wraps the C++ Core API object model
+- Disabled documentation generation by default in make to speed up development
+- Implemented CMake build-chain for ASE
+- Organized sample directory
+- Increased test coverage
+- Added Error API
+- Added SUSE support
+- Cleaned up dead legacy code
+- Various bug fixes
+
+Notes / Known Issues
+--------------------
+
+-   Seldom in stress tests, kernel panic may be encountered in kernel version 3.10. Preliminary debug information seems to indicate it may be related to hugepage support in the Linux kernel
```

Comment:
missing period. Other notes have periods. Do we end notes with periods or no? I'm not sure what the best way to end notes is.